src/CoreBundle/Resources/public/themes/standard/css/global.css

```diff
@@ -5,22 +5,68 @@ html {
 }
 
 body {
-    height: 100%;
+    overflow-x: hidden;
 }
 
 /* overwrite min-height: 100vh; set from coreui */
 body.app {
     min-height: 100%;
 }
 
+#cmscontainer .main {
+    display: flex;
+    flex-direction: column;
+}
+
+#cmscontainer .main .content {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+}
+
+#cmscontainer .main .content #tableEditorContainer {
+    flex: 1;
+}
+
+#cmscontainer .main .content #tableEditorContainer .tableeditcontainer {
+    height: 100%;
+}
+
+#cmscontainer .main .content #tableEditorContainer .tableeditcontainer #cmseditform {
+    height: 100%;
+}
+
+#cmscontainer .main .content #tableEditorContainer .tableeditcontainer #cmseditform #tabs-wrapper {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+}
+
+#cmscontainer .main .content #tableEditorContainer .tableeditcontainer #cmseditform #tabs-wrapper .tab-content {
+    flex: 1;
+}
+
+#cmsbreadcrumb .breadcrumb {
+    border-radius: 0;
+    margin-bottom: 0; /* overwrite bootstrap */
+}
+
+#cmsbreadcrumb .breadcrumb i {
+    color: #098dba;
+}
+
 a:hover {
     text-decoration: underline;
 }
 
 iframe {
+    width: 100%;
```

Comment:
I would not set every iframe to the exact same size. This would lead to sideeffects with custom modules.
Could you set this to an extra CSS class or define it more precise like only iframes inside of an table editor container?

Reply:
This only means that it will fill the containing element.

(And can/would be overwritten by more specific css if desired.)

Reply:
The 100% is o.k. it is more the height of 470px that makes me nervous.

Reply:
We will try if this height makes any problems.